### PR TITLE
Made models to accept data from TexasRescueMap.com/muckmap

### PR DESCRIPTION
**Pull request by Xiupan**
No controllers or views made yet though.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,8 @@
 Rails.application.routes.draw do
+  get 'volunteers/index'
```
Comment (ironcoconut) [inline config/routes.rb]: Should you use Resourceful routing? If there are going to be show, update, etc actions you should.

File: db/migrate/20170902213139_create_mucked_homes.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreateMuckedHomes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :mucked_homes do |t|
+      t.string :name
+      t.text :description
+      t.string :phone
+      t.string :pin
+      t.string :lat
```
Comment (ironcoconut) [inline db/migrate/20170902213139_create_mucked_homes.rb]: To keep it consistent in the DB, lat and lng should be written out, latitude and longitude. They should be floats not strings.

**Review by ironcoconut — CHANGES_REQUESTED** [review]
Good start. 

Don't forget to remove unused files such as stylesheets, helpers, etc.

File: db/schema.rb [hunk 3]
```diff
@@ -45,6 +45,17 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "mucked_homes", force: :cascade do |t|
+    t.string "name"
+    t.text "description"
+    t.string "phone"
+    t.string "pin"
+    t.string "lat"
```
Comment (jwo) [inline db/schema.rb]: lat/lng should be floats if we're going to be able to show distance by geo

Also let's go with `latitude` and `longitude`. We can output them as `lat/lng` if the API demands

File: app/controllers/mucked_homes_controller.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+class MuckedHomesController < ApplicationController
+
+  def index
```
Comment (jwo) [inline app/controllers/mucked_homes_controller.rb]: I'm not sure what the purpose of this and the VolunteersController are. Let's remove unless we really truly want them

File: test/fixtures/mucked_homes.yml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+# This model initially had no columns defined. If you add columns to the
+# model remove the '{}' from the fixture names and add the columns immediately
+# below each fixture, per the syntax in the comments below
+#
+one: {}
```
Comment (jwo) [inline test/fixtures/mucked_homes.yml]: We use fixtures pretty extensively in the app; let's have 2 filled out

File: test/fixtures/volunteers.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (jwo) [inline test/fixtures/volunteers.yml]: We use fixtures pretty extensively in the app; let's have 2 filled out

**Review by jwo — CHANGES_REQUESTED** [review]
@Xiupan Let me know if these changes make sense. If you need some help making them happen I can clone make the changes happen

File: db/schema.rb (hunk 3 above)
Comment (Xiupan) [reply db/schema.rb]: I think the problem is that they are already Strings on the TexasRescueMap MongoDB. But yeah, I guess we can convert them as we import the data?

File: config/routes.rb (hunk 1 above)
Comment (Xiupan) [reply config/routes.rb]: Okie dokie!

File: test/fixtures/mucked_homes.yml (hunk 5 above)
Comment (Xiupan) [reply test/fixtures/mucked_homes.yml]: I don't even know what fixtures are :x These were generated when I created the model I think? I'll take a look at existing fixtures and see if I can figure out what to do with the new ones...

File: app/controllers/mucked_homes_controller.rb (hunk 4 above)
Comment (Xiupan) [reply app/controllers/mucked_homes_controller.rb]: Okie dokie

**Comment by Xiupan** [comment]
Okie dokie, I think I did all the requested changes right this time :o

**Comment by jwo** [comment]
@Xiupan I pushed a commit to this pull-request:

* Renamed the fixture to by latitude/longitude (this is what made the tests fail)
* Fixed the latitude / longitude transposition in the fixtures
* Added a model test for both volunteer and mucked_home to sort by distance 
* Removed the controllers / views since we're not using them.

**Review by jwo — APPROVED** [review]

**Comment by jwo** [comment]
@ironcoconut Would you check the changed and click "approved changes" on this review?

Obvs if you still have concerns we can address :)

**Review by ironcoconut — APPROVED** [review]
Looks great. Thank you!

**Pull request merged**